client/gutenberg/extensions/jetpack/tiled-gallery/components/tiled-gallery-item.js

```diff
@@ -0,0 +1,101 @@
+
+// Since this is a Jetpack originated block in Calypso codebase,
+// we're relaxing className rules.
```

Comment:
I think that's okay for a first iteration, but in the long run I'd expect that we use a consistent code style with the rest of Calypso - what do you think?

Reply:
Yep, I think so but I'd leave that for another PR.

In the CSS that I grabbed from Jetpack, we might also have a bunch of stuff the block doesn't need.